build.sbt

```diff
@@ -88,7 +111,8 @@ lazy val util = project
   .settings(moduleName := "catbird-util")
   .settings(allSettings)
   .settings(
-    libraryDependencies += "com.twitter" %% "util-core" % utilVersion,
+    libraryDependencies +=
+      ("com.twitter" %% "util-core" % utilVersion).cross(CrossVersion.for3Use2_13),
```

Comment:
Though `util-core` is published for Scala 3, `finagle` is dependent on it, so we were forced to use the `for3Use2_13` option 🤷🏻‍♂️